app/views/static/metadata.yml

```diff
@@ -1,5 +1,5 @@
 # Title of a work -- this brief string is displayed prominently to end users
-title: '1918'
+title: 'doc set experiment'
```

Comment:
I'd prefer '1918' here, as it seemed to fit the hypothetical example better.  Could we restore it and make the document sets line consistent?  (Something like `['Diaries', 'Public`]`?)

Reply:
fixed.